routes/recordOverview.js

```diff
@@ -10,27 +38,131 @@ var route = function (req, res, next) {
         model: req.linz.model,
         record: clone(req.linz.record.toObject({ virtuals: true})),
         permissions: req.linz.model.linz.formtools.permissions,
-        formtools: req.linz.model.linz.formtools
+        formtools: req.linz.model.linz.formtools,
+        tabs: []
     };
 
     async.series([
 
         function (cb) {
 
-            req.linz.model.linz.formtools.overview.summary.renderer(req.linz.record, req.linz.model, function (err, content) {
+            var details = req.linz.model.linz.formtools.overview.details;
 
-                if (err) {
-                    return cb(err);
-                }
+            //render default overview if details is not provided or details provided is not a function OR not an Array containing fileds
+            if ( !details || !(typeof details === 'function' || (Array.isArray(details) && details.length)) ) {
+
```

Comment:
Shouldnn't `details` always exists? If user hasn't defined `overview.details` linz should default it to `['dateCreated','dateModified']`